src/briefcase/integrations/android_sdk.py

```diff
@@ -320,7 +320,7 @@ def devices(self):
                         details[key] = value
 
                     if parts[1] == "device":
-                        name = details["device"]
+                        name = details["model"]
```

Comment:
Really minor possible cosmetic improvement - would replacing `_` with a literal space make sense here? We have a limited sample size, but displaying a user-visible name of `Pixel 3a`/`Kogan Agora 9` seems preferable to `Pixel_3a`/`Kogan_Agora_9`. We're not using this name as an identifier (AFAIK?), so it should be purely cosmetic change?

Reply:
Sure :)